x-pack/test/accessibility/apps/spaces.ts

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// a11y tests for spaces, space selection and spacce creation and feature controls
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'spaceSelector', 'home', 'header', 'security']);
+  const a11y = getService('a11y');
+  const browser = getService('browser');
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+  const toasts = getService('toasts');
+
+  describe('Kibana spaces page meets a11y validations', () => {
+    before(async () => {
+      await esArchiver.load('empty_kibana');
+      await PageObjects.common.navigateToApp('home');
+    });
+
+    it('navigate to manage spaces home page', async () => {
+      await PageObjects.spaceSelector.openSpacesNav();
+      await retry.waitFor(
+        'Manage spaces option visible',
+        async () => await testSubjects.exists('manageSpaces')
+      );
+      await PageObjects.spaceSelector.clickManageSpaces();
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on create space button', async () => {
+      await PageObjects.spaceSelector.clickCreateSpace();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on input space name', async () => {
+      await PageObjects.spaceSelector.clickEnterSpaceName();
+      await a11y.testAppSnapshot();
+    });
+
+    it('add in space name as space a', async () => {
+      await PageObjects.spaceSelector.addSpaceName('space_a');
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on space avatar to open customize space avatar', async () => {
+      await toasts.dismissAllToasts(); // must clear toasts because they are not accessible
+      await PageObjects.spaceSelector.clickSpaceAcustomAvatar();
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on space initials input in ', async () => {
+      await PageObjects.spaceSelector.clickSpaceInitials();
+      await a11y.testAppSnapshot();
+    });
+
+    it('add in space initials as sa', async () => {
+      await PageObjects.spaceSelector.addSpaceInitials('sa');
+      await a11y.testAppSnapshot();
+    });
```

Comment:
I'm not clear on why these 2 tests above would be separate tests.  When you 'click on space initials input in ' aren't all the accessibility checks the same as after you type two characters into the field in 'add in space initials as sa'?